src/CanvasChart.js

```diff
@@ -15,12 +16,15 @@ const MARGIN = 4;
 const LINE_WIDTH = 1;
 const CORNER_RADIUS = 3;
 const AVATAR_SIZE = 24;
+const MILESTONE_SIZE = 24;
 
 const HEADER_HEIGHT = 20;
 const TASK_BAR_HEIGHT = 8;
 const TASK_ROW_HEIGHT =
   MARGIN + AVATAR_SIZE + MARGIN + TASK_BAR_HEIGHT + MARGIN;
 
+const MILESTONE_COLOR = "rgba(0,0,0)";
```

Comment:
Maybe it would be nice to use the color util (color for name string) for milestones too? :)

Reply:
Sorry, I went surfing. 🏄 Catching up. Let me know when I can drop back in, in fact!

Milestones are indeed zero length but I didn't want to punish extra data in particular. Yes to the color utility, though milestones tend to just present as black. But color util seems best.

Love this project!

Reply:
Actually, not sure that milestones should have colors; so, both options are supported now.

Reply:
I wasn't sure if milestones should always be black or have parent colors; so, the option to do either arrives in #46f6a65.